data/templates/esx-ks

```diff
@@ -114,9 +149,24 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
         esxcli network ip interface ipv6 address add -i <%=vmkname%> -I <%=n.ipv6.ipAddr%>
       <% } %>
     <% } %>
+    <% if( (undefined === n.ipv6) && (undefined === n.ipv4) ) { %>
+      <% vmkname = 'vmk' + vmkid++ %>
+      esxcli network vswitch standard portgroup add -p <%=n.device%> -v <%= typeof n.esxSwitchName!='undefined' ? n.esxSwitchName : 'vSwitch0' %>
+      esxcli network ip interface remove -i <%=vmkname%>
+      esxcli network ip interface add -i <%=vmkname%> -p <%=n.device%>
+      esxcli network ip interface ipv4 set -i <%=vmkname%> -t dhcp
+    <% } %>
   <% }); %>
 <% } %>
 
+<% if( typeof postInstallCommands !== 'undefined' ) { %>
+  <% postInstallCommands.forEach(function(n) { %>
+    <%=n%>
+  <% }); %>
+<% } %>
+
+#signify ORA the installation completed
+/usr/bin/wget --spider http://<%=server%>:<%=port%>/api/common/templates/renasar-ansible.pub
```

Comment:
see 'renasar' here, maybe need another cleanup PR to rename it to 'rachhd' @RackHD/corecommitters